sb3_contrib/common/maskable/algorithms/on_policy.py

```diff
@@ -0,0 +1,132 @@
+import gym
+import numpy as np
+import torch as th
+from stable_baselines3.common import logger
+from stable_baselines3.common.buffers import RolloutBuffer
+from stable_baselines3.common.callbacks import BaseCallback
+from stable_baselines3.common.on_policy_algorithm import OnPolicyAlgorithm
+from stable_baselines3.common.vec_env import VecEnv, is_vecenv_wrapped
+
+from sb3_contrib.common.maskable.algorithms.base import MaskableAlgorithm
+from sb3_contrib.common.maskable.buffers import MaskableRolloutBuffer
+from sb3_contrib.common.maskable.policies import MaskablePolicy
+from sb3_contrib.common.vec_env.wrappers import VecActionMasker
+
+
+class MaskableOnPolicyAlgorithm(MaskableAlgorithm, OnPolicyAlgorithm):
```

Comment:
same remark as before, this can be done directly in PPO

Reply:
Same response as above, is it better to design for future extension?